src/tiles/list/tile-list-compact.ts

```diff
@@ -1,22 +1,204 @@
 import { css, html, LitElement } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
+import { SortParam } from '@internetarchive/search-service';
+import DOMPurify from 'dompurify';
 import { TileModel } from '../../models';
+import { formatCount, NumberFormat } from '../../utils/format-count';
+import { formatDate, DateFormat } from '../../utils/format-date';
+import '../../mediatype-icon';
 
 @customElement('tile-list-compact')
 export class TileListCompact extends LitElement {
   @property({ type: Object }) model?: TileModel;
 
+  @property({ type: String }) baseNavigationUrl?: string;
+
+  @property({ type: Number }) currentWidth?: number;
+
+  @property({ type: Number }) currentHeight?: number;
+
+  @property({ type: Object }) sortParam?: SortParam;
+
   render() {
     return html`
-      <h1>${this.model?.title}</h1>
-      <h2>${this.model?.datePublished?.toDateString()}</h2>
+      <div id="list-line" class="${this.classSize}">
+        <div id="thumb">${this.img()}</div>
+        <div id="title">${DOMPurify.sanitize(this.model?.title ?? '')}</div>
+        <div id="date">${formatDate(this.date, this.formatSize)}</div>
+        <div id="creator">${DOMPurify.sanitize(this.model?.creator ?? '')}</div>
+        <div id="views">
+          ${formatCount(this.model?.viewCount ?? 0, this.formatSize)}
+        </div>
+        <div id="icon">
+          <mediatype-icon .mediatype=${this.model?.mediatype}></mediatype-icon>
+        </div>
+      </div>
     `;
   }
 
+  private img() {
+    if (this.model?.identifier) {
+      return html` <img
+        src="${this.baseNavigationUrl}/services/img/${this.model.identifier}"
+        alt="${this.model.identifier}"
+        class=${this.model?.mediatype}
+      />`;
+    }
+    return html``;
```

Comment:
If you're returning an empty template like this, it's better to use the `nothing` keyword so lit doesn't do any bindings. ie:
```
import { nothing } from 'lit';
...
return nothing;
```

Reply:
thanks, done